dnfdragora/const.py

```diff
@@ -134,14 +134,15 @@
 }
 
 RPM_ACTIONS = {
+    '10': _("Running scriptlet: %s"),
```

Comment:
I've added the new identifiers from the DNF documentation into dnfdaemon in manatools/dnfdaemon@1f8d65b1a90190f755cb144ae05775fe0384c45f

Please only use them as needed, as I'm not quite sure what they all do yet...